base/message_loop/message_pump.cc

```diff
@@ -54,6 +56,8 @@ std::unique_ptr<MessagePump> MessagePump::Create(MessagePumpType type) {
       // TODO(abarth): Figure out if we need this.
       NOTREACHED();
       return nullptr;
+#elif BUILDFLAG(IS_STARBOARD)
+      return std::make_unique<MessagePumpUIStarboard>();
 #else
       return std::make_unique<MessagePumpForUI>();
```

Comment:
I think we should be going through this more general class name instead as most platforms do (see `base/message_loop/message_pump_for_ui.h`)

Edit: actually it looks like you're already doing that below. Are you sure you need to add this?